src/resources/dicts/pt-br.edn

```diff
@@ -1,655 +1,838 @@
-{:on-boarding/demo-graph "Esse é um grafo de demonstração, mudanças não serão salvas enquanto uma pasta local não for aberta."
- :on-boarding/add-graph "Adicionar grafo"
- :on-boarding/open-local-dir "Abrir pasta local"
- :on-boarding/new-graph-desc-1 "Logseq funciona com Markdown e Org-mode. Você pode abrir uma pasta existente ou criar uma nova em seu dispositivo. Seus dados serão armazenados apenas neste dispositivo."
- :on-boarding/new-graph-desc-2 "Após abrir sua pasta, três pastas serão criadas nela:"
- :on-boarding/new-graph-desc-3 "/journals - armazena suas páginas diárias"
- :on-boarding/new-graph-desc-4 "/pages - armazena as outras páginas"
- :on-boarding/new-graph-desc-5 "/logseq - armazena configuração, custom.css e metadados."
- :help/start "Começar a usar"
- :help/about "Sobre o Logseq"
- :help/roadmap "Plano de implementação"
- :help/bug "Relato de erros"
- :help/feature "Pedido de funcionalidades"
- :help/changelog "Registo de alterações"
- :help/blog "Blog do Logseq"
- :help/docs "Documentação"
- :help/privacy "Política de privacidade"
- :help/terms "Termos"
- :help/awesome-logseq "Incrível Logseq"
- :help/shortcuts "Atalhos de Teclado"
+{:accessibility/skip-to-main-content "Pular para o conteúdo principal"
+ :tutorial/text #resource "tutoriais/tutorial-pt-br.md"
```

Comment:
```suggestion
 :tutorial/text #resource "tutorials/tutorial-pt-br.md"
```
@cosmo9able Looks like the lint is failing because of these misspelled paths

Reply:
Sorry, I made the correction.

Reply:
No worries. There's still another correction to be made. When you're ready, please ping Taylor for another review